### ISSUE TITLE
Tekken 7 - No audio devices found (389730)

### ISSUE THREAD

**Issue by davernos** [ValveSoftware/Proton#1075]
Two days ago it worked. Now its complaining about not finding audio devices. System wide I have no problem with the sound system, even the opening cinematics when launching the game have sound. When you reach the main menu, you get that message:

![389730_screenshots_20180831172941_1](https://user-images.githubusercontent.com/16683853/44921739-afe16b00-ad43-11e8-97ad-fb91497c3e8d.jpg)

"No audio devices found. Starting game without audio."

**Comment by kisak-valve** [comment]
Hello @davernos, please copy your system information from steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by davernos** [comment]
Gist: https://gist.github.com/davernos/0e817d8ff86a6825e09d36aed8e6182f
Just tell me if you need anything in english.

**Comment by davernos** [comment]
Yesterday, before the Season 2 patch launched, tried again (I've noticed those little updates on the Tekken 7 client) but the problem still happens.

**Comment by jonlap** [comment]
I had similar problem with wine (sound in opening cinematics but not in game). I had multiple audio devices which confused Tekken 7 somehow, iirc it was happening for some Windows users too. Disabled all other audio devices but one and it started working. I see you have only one sound device so probably not related, but maybe worth mentioning.

**Comment by davernos** [comment]
Now its on "Validating" loop since yesterday. Can't even launch the game.
![validating_tekken7](https://user-images.githubusercontent.com/16683853/45229110-4920fc00-b2c5-11e8-8838-81a4c287575c.png)

**Comment by davernos** [comment]
One thing that bothers me it's that Tekken 7 is getting little updates every now and then, according to Steam. Most of them don't even download anything, at least the bandwidth and disk usage shows 0 on both of them. On every reboot, it sets itself on queue for updates and gets "ready to play" in no time.

**Comment by davernos** [comment]
Could this be related to https://github.com/ValveSoftware/Proton/issues/54 ? They mention UE3, although Tekken 7 is UE4.

**Comment by jonlap** [comment]
@davernos I don't think that is the reason here, after all its officially supported game and works without any extra effort for most people. Could you try to run it with PROTON_LOG=1 %command% and show us the logs?

**Comment by davernos** [comment]
Added to the launch options but it's not generating any log. What's the default location or log name format? To search for it.

**Comment by jonlap** [comment]
@davernos there should be steam-389730.log in your home directory

**Comment by davernos** [comment]
[steam-389730.zip](https://github.com/ValveSoftware/Proton/files/2395597/steam-389730.zip)
Here's the log.

**Comment by davernos** [comment]
For some reason, switching to HDMI audio (I have HDMI to my TV as a second monitor), loading the game, and going back to my analog speakers works.
